src/bun.js/bindings/BunProcess.cpp

```diff
@@ -276,7 +276,13 @@ JSC_DEFINE_HOST_FUNCTION(Process_functionDlopen,
 
     WTF::String filename = callFrame->uncheckedArgument(1).toWTFString(globalObject);
     // Support embedded .node files
-    if (filename.startsWith("/$bunfs/"_s)) {
+    // See StandaloneModuleGraph.zig for what this "$bunfs" thing is
+#if OS(WINDOWS)
+#define StandaloneModuleGraph__base_path "B:/~BUN/"_s
```

Comment:
we probably need to check for backslashes as well if filename is coming from user

Reply:
ah actually maybe not if this is embedded